fastlane/Fastfile

```diff
@@ -377,6 +427,44 @@ end
     end
   end
 
+  #####################################################################################
+  # trigger_beta_build
+  # -----------------------------------------------------------------------------------
+  # This lane triggers a beta build on CI
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane trigger_beta_build [branch_to_build:<branch_name>]
+  #
+  #####################################################################################
+  lane :trigger_beta_build do | options |
+    circleci_trigger_job(
+      circle_ci_token: ENV["CIRCLE_CI_AUTH_TOKEN"], 
+      repository: REPOSITORY_NAME, 
+      organization: "Automattic",
+      branch: options[:branch_to_build],
```

Comment:
Not sure if that's an edge case worth covering given the use case for this workflow, but while testing I tried to run the lane before pushing the branch and the error I got in console was an unhelpful `[400] Bad Request`.

It might be worth adding a check at the release toolkit level to ensure the branch exists on the remote. I'd consider this a little improvement that can live in isolation, though.

Reply:
👍